openreview/venue_request/venue_request.py

```diff
@@ -504,8 +504,72 @@ def setup_decision_stage(self):
                 'required': False,
                 'description': 'Configure additional options in the decision form. Use lowercase for the field names and underscores to represent spaces. The UI will auto-format the names, for example: supplementary_material -> Supplementary Material. Valid JSON expected.'
             },
+            'decisions_file': {
+                'description': 'Upload a CSV file containing decisions for papers (one decision per line in the format: paper_id, decision, comment). Please do not add the column names as the first row',
+                'order': 37,
+                'value-file': {
+                    'fileTypes': ['csv'],
+                    'size': 50
+                },
+                'required': False
+            }
         }
 
+        decisions_upload_status_content = {
+            'title': {
+                'value': 'Decision Upload Status',
+                'required': True,
+                'order': 1
+            },
+            'decision_posted': {
+                'value-regex': '.*',
+                'description': 'No. of papers decision was posted for',
+                'required': True,
+                'markdown': True,
+                'order': 2
+            },
+            'error': {
+                'value-regex': '[\\S\\s]{0,200000}',
+                'description': 'List of papers whose decision were not posted due to an error',
+                'required': False,
+                'markdown': True,
+                'order': 5
+            },
+            'comment': {
+                'order': 6,
+                'value-regex': '[\\S\\s]{1,200000}',
+                'description': 'Your comment or reply (max 200000 characters).',
+                'required': False,
+                'markdown': True
+            }
+        }
+
+        with open(self.venue_request.decision_upload_status_process, 'r') as f:
+            file_content = f.read()
+            file_content = file_content.replace("GROUP_PREFIX = ''", "GROUP_PREFIX = '" + self.venue_request.super_user + "'")
+            self.venue_request.decision_upload_super_invitation = self.venue_request.client.post_invitation(
+                openreview.Invitation(
```

Comment:
So does this mean that existing venues won't have this feature? I got a process function error when trying this for MIDL: "The Invitation OpenReview.net/Support/-/Request265/Decision_Upload_Status was not found". Is there a way to enable it for already deployed venues?

Reply:
Yes, we will just have to create the invitations for `Decision_Upload_Status` for all venues once we deploy this.